File: Android/TP_HWE - Copy/app/src/main/java/Entities/News.java
package Entities;

import java.util.Date;
import java.util.List;

/**
 * Created by Me on 1/10/2019.
 */

public class News {
    private int Id;
    private String Title;
    private String Content;
    private Date LastModified;
    private List<Comment> Comments;
    private List<Picture> Pictures;
    private List<Audio> AudioRecordings;

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }

    public String getContent() {
        return Content;
    }

    public void setContent(String content) {
        Content = content;
    }

    public Date getLastModified() {
        return LastModified;
    }

    public void setLastModified(Date lastModified) {
        LastModified = lastModified;
    }

    public List<Comment> getComments() {
        return Comments;
    }

    public void setComments(List<Comment> comments) {
        Comments = comments;
    }

    public List<Picture> getPictures() {
        return Pictures;
    }

    public void setPictures(List<Picture> pictures) {
        Pictures = pictures;
    }

    public List<Audio> getAudioRecordings() {
        return AudioRecordings;
    }

    public void setAudioRecordings(List<Audio> audioRecordings) {
        AudioRecordings = audioRecordings;
    }
}

File: Baza/DataLayerLib/ObjectModel/Entities/News.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ObjectModel.Entities
{
    public class News
    {
        public virtual int Id { get; protected set; }
        public virtual string Title { get; set; }
        public virtual string Content { get; set; }
        public virtual DateTime LastModified { get; set; }
        public virtual Picture BackgroundPicture { get; set; }
        public virtual IList<NewsModified> Modifications { get; set; }
        public virtual IList<Picture> Pictures { get; set; }
        public virtual IList<Audio> AudioRecordings { get; set; }
        public virtual IList<Comment> Comments { get; set; }
        public News()
        {
            Modifications = new List<NewsModified>();
            Pictures = new List<Picture>();
            AudioRecordings = new List<Audio>();
            Comments = new List<Comment>();
        }
    }
}
